Gemfile

```diff
@@ -6,6 +6,7 @@ gem 'net_http_unix'
 gem 'netaddr'
 
 group :development do
+  gem 'rubocop', '0.34.2'
```

Comment:
Good catch. I was going off my installed version thinking it was the latest. This version was released in the last 7 days. Probably should update.